docs/requirements.txt

```diff
@@ -7,7 +7,7 @@ docutils==0.18.1
 idna==2.10
 imagesize==1.2.0
 importlib-metadata==4.6.1
-Jinja2==3.0.1

```

Comment:
Breaks docs, update sphinx as well 
"""
Could not import extension sphinx.builders.latex (exception: cannot import name 'contextfunction' from 'jinja2' (/home/docs/checkouts/readthedocs.org/user_builds/deepgnn/envs/249/lib/python3.9/site-packages/jinja2/__init__.py))
"""